src/libs/actions/PaymentMethods.js

```diff
@@ -256,6 +256,7 @@ function transferWalletBalance(paymentMethod) {
                 value: {
                     loading: true,
                     error: null,
+                    errors: null,
```

Comment:
Reviewing a bit late here hehe, why are we having `error` and `errors`? Shouldn't we have just one property for error (or errors)?

Reply:
We will deploy support both formats on App first. After the Web-PR is live and only sending data under the `errors` key, we can get rid of the `error` key.